internal/build/container_updater.go

```diff
@@ -25,21 +25,29 @@ func NewContainerUpdater(dCli docker.Client) *ContainerUpdater {
 func (r *ContainerUpdater) UpdateInContainer(ctx context.Context, cID container.ID, paths []PathMapping, filter model.PathMatcher, runs []model.Cmd, hotReload bool, w io.Writer) error {
 	span, ctx := opentracing.StartSpanFromContext(ctx, "daemon-UpdateInContainer")
 	defer span.Finish()
+	l := logger.Get(ctx)
 
 	// rm files from container
-	toRemove, err := MissingLocalPaths(ctx, paths)
+	toRemove, toArchive, err := MissingLocalPaths(ctx, paths)
 	if err != nil {
 		return errors.Wrap(err, "MissingLocalPaths")
 	}
 
+	if len(toRemove) > 0 {
+		l.Verbosef("Deleting %d file(s) from container: %s", len(toRemove), cID.ShortStr())
```

Comment:
Why verbose? This seems like good info to have in general. If the concern is length, I think there are ways to make it simpler.

In-container updating is fraught, and I'd rather overexplain (esp given comments we get like "I'm not sure sure what is happening or why")